Fixing pip installation

fmalina opened the pull request:
Updating the manifest didn't solve the installation problem.
This solution is not DRY, but better be safe as I can't easily test this myself.

More information:
http://stackoverflow.com/questions/14399534/how-can-i-reference-requirements-txt-for-the-install-requires-kwarg-in-setuptool

Error in question:

$.. pip3 install django-openinghours
Collecting django-openinghours
  Using cached django-openinghours-0.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/4m/q0mv284n1_qgm7ctvkrmgn4c0000gp/T/pip-build-osj2dtv8/django-openinghours/setup.py", line 18, in <module>
        install_requires=open('requirements.txt').read().split(),
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'

Commit:
Fixing pip installation

Updating the manifest didn't solve the installation problem.
This solution is not DRY, but better be safe as I can't easily test this myself.

More information:
http://stackoverflow.com/questions/14399534/how-can-i-reference-requirements-txt-for-the-install-requires-kwarg-in-setuptool

Error in question:

$.. pip3 install django-openinghours
Collecting django-openinghours
  Using cached django-openinghours-0.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/4m/q0mv284n1_qgm7ctvkrmgn4c0000gp/T/pip-build-osj2dtv8/django-openinghours/setup.py", line 18, in <module>
        install_requires=open('requirements.txt').read().split(),
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'

jvamvas commented:
@fmalina I agree that in this case the DRY violation can be justified. Thank you!

fmalina commented:
Can you push this fix to PyPI please?

pip install is still failing.
```

@etc2 ~: pip3 install django-openinghours
Downloading/unpacking django-openinghours
  Downloading django-openinghours-0.1.1.tar.gz
  Running setup.py (path:/tmp/pip_build_f/django-openinghours/setup.py) egg_info for package django-openinghours
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_f/django-openinghours/setup.py", line 18, in <module>
        install_requires=open('requirements.txt').read().split(),
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_f/django-openinghours/setup.py", line 18, in <module>

    install_requires=open('requirements.txt').read().split(),

FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_f/django-openinghours
Storing debug log for failure in /tmp/tmpsbsyyj2z
```

Thank you.

jvamvas commented:
@fmalina I have now released 0.1.2.

camflan commented:
@kingsofteng I was successful just now,
```
Successfully installed django-compat-1.0.15 django-openinghours-0.1.5 django-threadlocals-0.8
```
